cmd/dbaas_user_show_kafka.go

```diff
@@ -0,0 +1,56 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/exoscale/cli/pkg/account"
+	"github.com/exoscale/cli/pkg/globalstate"
+	"github.com/exoscale/cli/pkg/output"
+	"github.com/exoscale/cli/table"
+	v3 "github.com/exoscale/egoscale/v3"
+)
+
+type dbaasKafkaUserShowOutput struct {
+	AccessKey        string    `json:"access-key,omitempty"`
+	AccessCert       string    `json:"access-cert,omitempty"`
+	AccessCertExpiry time.Time `json:"access-cert-expiry,omitempty"`
+}
+
+func (o *dbaasKafkaUserShowOutput) formatUser(t *table.Table) {
+	t.Append([]string{"Access Key", o.AccessKey})
+	t.Append([]string{"Access Cert", o.AccessCert})
+	t.Append([]string{"Access Cert Expiry", o.AccessCertExpiry.String()})
+}
+
+func (c *dbaasUserShowCmd) showKafka(ctx context.Context) (output.Outputter, error) {
+
+	client, err := switchClientZoneV3(ctx, globalstate.EgoscaleV3Client, v3.ZoneName(account.CurrentAccount.DefaultZone))
+	if err != nil {
+		return &dbaasUserShowOutput{}, err
+	}
+
+	s, err := client.GetDBAASServiceKafka(ctx, c.Name)
+	if err != nil {
+		return &dbaasUserShowOutput{}, err
+	}
+
+	for _, u := range s.Users {
+		if u.Username == c.Username {
+			return &dbaasUserShowOutput{
+				Username: c.Username,
+				Password: u.Password,
```

Comment:
I don't think this should be here. In general all secrets should not be returned by the "show SERVICE user" commands. While as per today you might find the information you want in them, they will be removed, and you would end up showing empty strings. There are specific reveal-secret commands afaik.
Or maybe i did misunderstand, but in this case maybe we could review together in a call?

Reply:
Your understanding is correct, this was indeed the password as returned by Aiven's API and it was shown as is. Removed from all `show` commands.

Reply:
Added a command `reveal-secrets` (3e80a064) specifically to display passwords and other secrets